Makefile

```diff
@@ -2,13 +2,15 @@ GO ?= go
 GOFMT ?= gofmt
 AWK ?= awk
 
-COLLECTOR_VERSION = $(shell grep CollectorVersion common/metadata.go | $(AWK) '{gsub(/"/, "", $$3); print $$3}')
+COLLECTOR_VERSION = $(shell jq -r ".version" versions.json)
```

Comment:
Not really happy about depending on `jq` here. How about using a simple makefile snippet that we include in the Makefile and that can be parsed in the Ruby code?

version.mk:
```
COLLECTOR_VERSION = 0.1.0
COLLECTOR_VERSION_SUFFIX = -beta.4
COLLECTOR_REVISION = 1
```